docusaurus/docs/iOS/swiftui/components/read-indicators.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Read Indicators
+---
+
+## Read Indicators Overview
+
+The SwiftUI SDK supports read indicators, which indicate whether a message was read by the other channel participants. 
+
+The read indicators are available in both the channel and the message list. The default implementation shows one gray checkmark if a message was sent, but not read by anyone and two blue (depending on tint color) checkmarks for a read message. If a message is sent in a group, the number of readers of the message is also shown.
+
+It is possible to customize the read indicator in the message list. For example, some messaging apps show small icons of the users who have read the message. 
+
+In order to implement your own version of the read indicator, you will need to implement the `makeMessageReadIndicatorView` in the `ViewFactory` protocol.
+
+```swift
+public func makeMessageReadIndicatorView(
+    channel: ChatChannel,
+    message: ChatMessage,
+    showReadCount: Bool
+) -> some View {
+	CustomMessageReadIndicatorView(
+		channel: ChatChannel,
+    	message: ChatMessage
+	)
+}
+```
+
+In this method, you receive the channel and the message as parameters. Additionally, you receive the `showReadCount`, which you can ignore in case you don't want to support conditional display of the number of users that read a message.
```

Comment:
One thing that is unclear to me is how I can configure whether to show the read count. So, specifically the `showReadCount` value. In `Utils`?

Would be nice to add to the documentation as well. :)

Reply:
Basically it's just the value passed in the method. It's up to the custom view implementation to decide whether to show it or not (based on this value). When I think about it, we might even get rid of it.

Reply:
Removed the parameter, since it's actually implementation detail.